opm/simulators/wells/MSWellHelpers.cpp

```diff
@@ -235,31 +234,28 @@ ValueType calculateFrictionFactor(const double area, const double diameter,
         return 0.0;
     }
 
```

Comment:
ideally, the code above 
```c++
    if ( re == 0.0 ) {
        // make sure it is because the mass rate is zero
        assert(w == 0.);
        return 0.0;
    }
```
can be removed. Will investigate a little. Because removing it slows down a little bit running of the case we are using.